### PR TITLE
update information in About Us

### PR DESCRIPTION
**Pull request by Livian1107**

### CODE REVIEW

**Review by kohyeeru — CHANGES_REQUESTED** [review]
Please leave this PR open for changes until we have finalized all of our roles in the team from a group discussion, thank you. :>

File: docs/AboutUs.adoc [hunk 1]
```diff
@@ -3,7 +3,7 @@
 :imagesDir: images
 :stylesDir: stylesheets
 
-AddressBook - Level 4 was developed by the https://se-edu.github.io/docs/Team.html[se-edu] team. +
+AddressBook - Level 4 was developed by the https://se-edu.github.io/docs/Team.html[Team T09-B3] team. +
```
Comment (kohyeeru) [inline docs/AboutUs.adoc]: Do we need to leave this be as a credit to the original creators? I don't see any mentions of required changes to this part in the module website. @AdityaA1998 @Livian1107 @EdwardKSG

File: docs/AboutUs.adoc [hunk 2]
```diff
@@ -27,12 +27,12 @@ Responsibilities: UI
...
 '''
 
-=== Johnny Doe
-image::yijinl.jpg[width="150", align="left"]
-{empty}[http://github.com/yijinl[github]] [<<johndoe#, portfolio>>]
+=== Lai Liwen
+image::liwen.jpg[width="150", align="left"]
+{empty}[https://github.com/Livian1107[github]] [<<johndoe#, portfolio>>]
 
 Role: Developer +
```
Comment (kohyeeru) [inline docs/AboutUs.adoc:35]: Please update your roles and responsibilities as discussed in the team. :>

File: docs/AboutUs.adoc [hunk 3]
```diff
@@ -27,12 +27,12 @@ Responsibilities: UI
 
 '''
 
-=== Johnny Doe
-image::yijinl.jpg[width="150", align="left"]
-{empty}[http://github.com/yijinl[github]] [<<johndoe#, portfolio>>]
+=== Lai Liwen
+image::liwen.jpg[width="150", align="left"]
+{empty}[https://github.com/Livian1107[github]] [<<johndoe#, portfolio>>]
```
Comment (kohyeeru) [inline docs/AboutUs.adoc:33]: Do we need to update the reference for portfolio? @AdityaA1998 @Livian1107 @EdwardKSG

**Review by kohyeeru — CHANGES_REQUESTED** [review]
Some questions for the team to address, thank you!

File: docs/AboutUs.adoc (hunk 1 above)
Comment (AdityaA1998) [reply docs/AboutUs.adoc]: No. Infact we need to replace it with our teams description

File: docs/AboutUs.adoc (hunk 3 above)
Comment (AdityaA1998) [reply docs/AboutUs.adoc:33]: For now we can just let John Doe's portfolio be. Later when we make our own,  we can put the link over here

**Review by AdityaA1998 — APPROVED** [review]
Just read through the comments. Apart from that it seems okay to me

File: docs/AboutUs.adoc (hunk 1 above)
Comment (kohyeeru) [reply docs/AboutUs.adoc]: Ok, then that means the hyperlink to se-edu should be removed as well as the dummy line below?

Comment (AdityaA1998) [reply docs/AboutUs.adoc]: No. We have to change as per our project.

File: docs/AboutUs.adoc [hunk 4]
```diff
@@ -27,12 +27,12 @@ Responsibilities: UI
...
 
 Role: Developer +
-Responsibilities: Data
+Responsibilities: UI
```
Comment (AdityaA1998) [inline docs/AboutUs.adoc]: Is your role UI ? didn't we decide smth else

**Review by kohyeeru — APPROVED** [review]

File: docs/AboutUs.adoc [hunk 5]
```diff
@@ -3,8 +3,7 @@
 :imagesDir: images
 :stylesDir: stylesheets
 
-AddressBook - Level 4 was developed by the https://se-edu.github.io/docs/Team.html[se-edu] team. +
-_{The dummy content given below serves as a placeholder to be used by future forks of the project.}_ +
+AddressBook - Level 4 was developed by the https://github.com/CS2103JAN2018-T09-B3/main[T09-B3] team. +
```
Comment (AdityaA1998) [inline docs/AboutUs.adoc]: We need to write our project reflection instead of Addressbook lvl 4. Just one line but we need to add

**Pull request merged**